streampark-console/streampark-console-service/src/main/assembly/script/upgrade/pgsql/2.1.0.sql

```diff
@@ -54,12 +52,93 @@ comment on column "public"."t_yarn_queue"."modify_time" is 'modify time';
 
 alter table "public"."t_yarn_queue" add constraint "t_yarn_queue_pkey" primary key ("id");
 alter table "public"."t_yarn_queue" add constraint "unique_team_id_queue_label" unique("team_id", "queue_label");
-alter table "public"."t_flink_log" add column "option_name" int2;
-alter table "public"."t_user" add column "login_type" int2 default 0;
-ALTER TABLE public.t_flink_app ALTER COLUMN state TYPE int4 USING state::int4;
+
 
 drop table if exists "public"."t_flink_tutorial";
 
+-- type change
+alter table "public"."t_app_backup" modify column "path" varchar(128) collate utf8mb4_general_ci default null;
```

Comment:
Modify to `alter table "public"."t_app_backup" alter column "path" TYPE varchar(128) collate "pg_catalog"."default";`